singer_sdk/testing/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""Tools and standard tests for Tap/Target implementations."""
+
+from .config import SuiteConfig
+from .factory import get_tap_test_class, get_target_test_class, get_test_class
+from .legacy import (
+    _get_tap_catalog,
+    _select_all,
+    get_standard_tap_tests,
```

Comment:
Just want to confirm that this makes it backwards-compatible, i.e. this continues to work for folks that may update the SDK but continue using the legacy test suite:

```python
from singer_sdk.testing import get_standard_tap_tests
```

Reply:
Yes 🙂 I did a [spot-check](https://github.com/MeltanoLabs/tap-github/pull/177) on `tap-github` to reassure myself 😅

Reply:
Awesome, thanks for confirming!